### PR TITLE
New package: magic-wormhole-0.11.2

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/python3-hkdf/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-hkdf'
+pkgname=python3-hkdf
+version=0.0.3
+revision=1
+archs=noarch
+wrksrc=hkdf-${version}
```
Comment (Chocimier) [inline srcpkgs/python3-hkdf/template]: quote

File: srcpkgs/python3-hkdf/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+short_desc="HMAC Key Derivation module"
+maintainer="travankor <travankor@tuta.io>"
+license="BSD-2-Clause"
+homepage="https://pypi.org/project/hkdf/"
+distfiles="https://files.pythonhosted.org/packages/c3/be/327e072850db181ce56afd51e26ec7aa5659b18466c709fa5ea2548c935f/hkdf-${version}.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/python3-hkdf/template]: use PYPI_SITE, try hashless address like in python3-autobahn

File: srcpkgs/magic-wormhole/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+pycompile_module="wormhole"
+hostmakedepends="python3 python3-setuptools"
+depends="python3-autobahn python3-cffi python3-click python3-humanize
+ python3-idna python3-pynacl python3-service_identity python3-spake2
+ python3-tqdm python3-txtorcon"
+short_desc="Get things from one computer to another, safely"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://magic-wormhole.readthedocs.io/en/latest/"
+distfiles="https://github.com/warner/${pkgname}/archive/${version}.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/magic-wormhole/template]: replace ${pkgname} with magic-wormhole

File: srcpkgs/python3-hkdf/template (hunk 2 above)
Comment (travankor) [reply srcpkgs/python3-hkdf/template]: switched it to debian mirror for now

Comment (abenson) [reply srcpkgs/python3-hkdf/template]: "${PYPI_SITE}/h/hkdf/hkdf-${version}.tar.gz" works

File: srcpkgs/python3-spake2/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-spake2'
+pkgname=python3-spake2
+version=0.8
+revision=1
+archs=noarch
+wrksrc="python-spake2-${version}"
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="python3-hkdf"
+short_desc="Pure-python implementation of the SPAKE2 Password-Authenticated Key Exchange algorithm"
```
Comment (pullmoll) [inline srcpkgs/python3-spake2/template]: Too long. I'd suggest "SPAKE2 Password-Authenticated Key Exchange".
The prefix `python3-` and the `noarch` inherently tells it's pure python.

**Comment by travankor** [comment]
Ready for another review. (Thanks btw to Chocimier, Aedinius, and Pullmoll for their reviews.)

File: srcpkgs/python3-autobahn/template [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-autobahn'
+pkgname=python3-autobahn
+version=19.6.2
+revision=1
+archs=noarch
+wrksrc="autobahn-${version}"
+build_style=python3-module
+pycompile_module="wamp"
```
Comment (Chocimier) [inline srcpkgs/python3-autobahn/template]: pycompile_module="autobahn"

File: srcpkgs/python3-hkdf/template [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-hkdf'
+pkgname=python3-hkdf
+version=0.0.3
+revision=1
+archs=noarch
+wrksrc="hkdf-${version}"
+build_style=python3-module
```
Comment (Chocimier) [inline srcpkgs/python3-hkdf/template]: pycompile_module="hkdf.py"

File: srcpkgs/python3-spake2/template [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-spake2'
+pkgname=python3-spake2
+version=0.8
+revision=1
+archs=noarch
+wrksrc="python-spake2-${version}"
+build_style=python3-module
```
Comment (Chocimier) [inline srcpkgs/python3-spake2/template]: pycompile_module="spake2"

File: srcpkgs/python3-txaio/template [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-txaio'
+pkgname=python3-txaio
+version=18.8.1
+revision=1
+archs=noarch
+wrksrc="txaio-${version}"
+build_style=python3-module
```
Comment (Chocimier) [inline srcpkgs/python3-txaio/template]: pycompile_module="python3-txaio"

File: srcpkgs/python3-txtorcon/template [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-txtorcon'
+pkgname=python3-txtorcon
+version=19.0.0
+revision=1
+archs=noarch
+wrksrc="txtorcon-${version}"
+build_style=python3-module
```
Comment (Chocimier) [inline srcpkgs/python3-txtorcon/template]: pycompile_module="txtorcon"

**Comment by Chocimier** [comment]
python-cryptography is updated now.

**Comment by travankor** [comment]
sorry i messed this up while rebasing (and now github won't "reopen" the pull request)

**Pull request closed without merging**